### DISCUSSION TITLE
zWave USB

### DISCUSSION THREAD

**Discussion by calz64** [indomus/forum#1133] [HUB: Home Assistant]
Buongiorno avete una chiavetta zWave usb da consigliarmi testata con HomeAssistant-OS?
Grazie

**Comment by snakuzzo** [comment]
La Aeotek Stick Gen5.
Ecco la [NOSTRA RECENSIONE](https://indomus.it/componenti/recensione-aeotec-z-stick-gen5/)

**Reply by indomus** [reply]
Va molto bene anche la [ZWave.me UZB1](https://indomus.it/link/zwave-me-uzb1/)

**Reply by calz64** [reply]
ok, non conoscevo il canale.
la stick Gen5 però ha il problemino con raspy4 e la Gen5plus non la trovo facile da acquistare.
comunque grazie provo in chat

**Reply by indomus** [reply]
no la ZWave.me non dovrebbe aver problemi con raspi4

**Reply by calz64** [reply]
scusa ma ci siamo sovrapposti rispondevo a snakuzzo grazie per la dritta ho visto anche la Everspring - Controller USB Z-Wave+
quindi si gira su queste?

**Reply by indomus** [reply]
queste non l'abbiamo testate, sorry :/
